packages/@uppy/xhr-upload/src/index.ts

```diff
@@ -382,20 +472,20 @@ export default class XHRUpload extends BasePlugin {
 
         files.forEach((file) => {
           this.uppy.emit('upload-progress', file, {
-            uploader: this,
```

Comment:
Removed `uploader` as it's used nowhere.

Reply:
It might be used by users – it might have been added as a response to a user request. You also added `uploadStarted`?

Reply:
I can't be typed properly but maybe we should leave it was `any` and `ts-expect-error`. I added `uploadStarted` because it was a bug that it was not present.